src/SpellingBee/saveState.py

```diff
@@ -0,0 +1,102 @@
+# Authors: Yah'hymbey Baruti-Bey, Jacob Lovegren
+# Course : CSCI 420
+# Modified Date: 2/2/2023
+# State structure for puzzles
+
+import sqlite3
+import generateSubset
+
+class Puzzle:
+    
+    def __init__(self, keyLett, pangram):
+        self.keyLett = keyLett
+        self.pangram = pangram
```

Comment:
Double checking what we're storing here. I'm pretty sure PANGRAM is refering to the unique set of 7 letters for the game. Right now, we're scoring based on pangram in the DB, and don't need to worry about the base starting word after those words are derived. I would change pangram variable name to something like uniqueLetters